<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogNuevoUsuario</class>
 <widget class="QDialog" name="dialogNuevoUsuario">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Actualizar Usuario</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Primer Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_first_name">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Actualizar Usuario</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Segundo Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_second_name">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_username">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_password">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contrasena</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Telefono</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_phone">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_role">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>510</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>admin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>user</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>580</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_first_surname">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_second_surname">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Primer Apellido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Segundo Apellido</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
